torbrowser: add .desktop file

jgillich opened the pull request.

jgillich committed:
torbrowser: 5.0.6 -> 5.0.7, add desktop file

cillianderoiste commented on pkgs/tools/security/tor/torbrowser.nix:
```diff
@@ -57,15 +58,23 @@ in stdenv.mkDerivation rec {
     EOF
     chmod +x $out/bin/tor-browser
+
+    cat > "$out/share/applications/$name.desktop" << EOF
```
I think you can use makeDesktopItem here instead. That would be neater.

jgillich commented:
Sorry for the delay, now uses makeDesktopItem and I've updated it to 5.0.7 and added the correct icon. Still need to update the 32 bit hash, but how do I get it? @vcunat

vcunat commented:
I do it like this:
```
$ nix-build -A pkgsi686Linux.torbrowser.src
building path(s) ‘/nix/store/0a9pr1spzv9rsw74jdqgm44bx26sn1jg-tor-browser-linux32-5.0.7_en-US.tar.xz’
output path ‘/nix/store/0a9pr1spzv9rsw74jdqgm44bx26sn1jg-tor-browser-linux32-5.0.7_en-US.tar.xz’ should have sha256 hash ‘1q5mf91xxj1xs4ajj9i6mdhnzqycbdvprkzskx8pl6j9ll2hlsyh’, instead has ‘1401j340348rqwd5c1m8hnxw7qkwpzqfa64q01cp08lz9cxxml4r’
error: build of ‘/nix/store/x3f5mjfd54q212dylhw06xhvjd39lz2k-tor-browser-linux32-5.0.7_en-US.tar.xz.drv’ failed
```

jgillich commented:
Thanks! Should be good to go now.
